splink/cluster_studio.py

```diff
@@ -310,9 +320,17 @@ def render_splink_cluster_studio_html(
             else:
                 # Using sensible default for min_nodes. Might become option
                 # for users in future
-                cluster_ids = _get_lowest_density_cluster_ids(
-                    linker, _df_cluster_metrics, sample_size, min_nodes=3
+                cluster_ids = _get_lowest_density_clusters(
```

Comment:
Not important, and realise it was already like this, but could get rid of this else statement and de-indent, as the `if` exits. Personal pref though, so happy if you prefer to leave as-is!

Reply:
Good point. I prefer the else-less version too :)